docker/nginx-consul-template/nginx.conf.tmpl

```diff
@@ -0,0 +1,32 @@
+events {
+    worker_connections 4096;
+}
+
+http {
+{{- with $service_query := (printf "%s.%s" (env "DE_ENV") (env "NGINX_PROXY_SERVICE_NAME")) }}
+  {{- if service $service_query }}
+    upstream service {
+    {{- range service $service_query }}
+      server {{.Address}}:{{.Port}} max_fails=3 fail_timeout=60 weight=1 {{- if not (.Tags.Contains (key_or_default (printf "%s/%s/color" (env "DE_ENV") (env "NGINX_PROXY_SERVICE_NAME")) "green"))}} backup{{end}}; # {{.ID}} {{.Tags}}
```

Comment:
re: a. I think it sounds like there's more advantages of keeping it dumber; I'll let it just default to green arbitrarily like I have it now

re: b: Hm. We don't have a particularly built-up alerts infrastructure of any sort, which is what makes this harder, I think. The log will certainly include the "upstream has no servers" message from the `nginx -s reload` but I don't know that we have a way to propagate that to ourselves or anything. I think perhaps the thing to do is keep it how it is for now and pay attention when we start doing this, and change it once we have a better idea of what we need. It may be that in practice, this sort of misconfiguration will just always result in it becoming a force-502 eventually, since we'll eventually just take down the wrong-color servers anyway, and then we get the same quick notification from errors. So I think I'll leave it.

Thanks for your opinions on it!